src/GoodsList.tsx

```diff
@@ -8,7 +8,7 @@ type Props = {
 export const GoodsList: React.FC<Props> = ({ goods }) => (
   <ul>
     {goods.map(good => (
-      <li key={good.id} data-cy="good">
+      <li key={good.id} data-cy="good" style={{ color: `${good.color}` }}>
```

Comment:
You should destructure 'good' and also it's better to write each attribute on a blank line